.github/workflows/mediasoup-worker-prebuild.yaml

```diff
@@ -44,29 +46,36 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        if: startsWith(github.ref, "refs/tags/")
```

Comment:
Why not skipping the whole workflow instead? Skipping individual steps looks odd

Reply:
> Why not skipping the whole workflow instead?

How to do that? 😀
That's exactly what I've been trying to do.

Reply:
There is `if` at job level as well: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

But if job is only triggered by tag, this check doesn't have any effect in the first place.